### PR TITLE
Moved the period inside the quotation marks

### PR DESCRIPTION
**Pull request by nguyen-tina**
In the 'Tips for Applying' section, I moved the period from outside the quotation marks to inside.

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -45,8 +45,8 @@
   "tab0-list1-item1": "Everyone (UI and PUA applicants) uses the same form. <2>Register or log in with Benefit Programs Online</2> and click “File New Claim.”",
   "tab0-list1-item2": "File a new claim if you have already applied for UI but are eligible for PUA because you are a business owner, independent contractor, self-employed worker, freelancer, or gig worker affected by COVID-19, <strong>and</strong> you have <strong>not</strong> already appealed a $0 award notice, been asked to verify your identity, or requested a wage investigation.",
   "tab0-list1-item3": "If you are a small business owner, independent contractor, self-employed worker, freelancer, or gig worker, here’s how to answer key questions in the application form:",
-  "tab0-list1a-item1": "On the Employment History screen when you supply your last employer information, select \"No\".",
-  "tab0-list1a-item2": "On the Availability Information page, answer question 7 with \"No\".",
+  "tab0-list1a-item1": "On the Employment History screen when you supply your last employer information, select \"No.\",
```
Comment (kalvinwang) [inline public/locales/en/translation.json:48]:
```suggestion
  "tab0-list1a-item1": "On the Employment History screen when you supply your last employer information, select \"No.\"",
```

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -45,8 +45,8 @@
...
+  "tab0-list1a-item2": "On the Availability Information page, answer question 7 with \"No.\",
```
Comment (kalvinwang) [inline public/locales/en/translation.json:49]:
```suggestion
  "tab0-list1a-item2": "On the Availability Information page, answer question 7 with \"No.\"",
```

**Comment by nguyen-tina** [comment]
Thanks for the explanation! Hahah it's very comforting to hear that you're not familiar with what an error looks like in github 🤪 I've accepted your suggested and understand why I need to put a quotation mark at the end!

**Comment by kalvinwang** [comment]
@nguyen-tina yeah Sharon and I even have our code editors set up to automatically fix some errors :joy: 

(Note: I'll merge this PR right after I merge the 2x revert PR later today. That's where the conflict is from, because the translations are missing on master.)

**Pull request merged**